docs/Editing-Configuration-Files.md

```diff
@@ -89,9 +89,10 @@ Here is a sample of the three basic types: respectively Boolean, Number and Stri
  * **script-torrent-done-filename:** String (default = "") Path to script.
  * **script-torrent-done-seeding-enabled:** Boolean (default = false) Run a script when a torrent is done seeding. Environmental variables are passed in as detailed on the [Scripts](./Scripts.md) page
  * **script-torrent-done-seeding-filename:** String (default = "") Path to script.
- * **tcp-enabled:** Boolean (default = true) Optionally disable TCP connection to other peers. Never disable TCP when you also disable UTP, because then your client would not be able to communicate. Disabling TCP might also break webseeds. Unless you have a good reason, you should not set this to false.
+ * **tcp-enabled:** Boolean (default = true) Optionally disable TCP connection to other peers. Never disable TCP when you also disable µTP, because then your client would not be able to communicate. Disabling TCP might also break webseeds. Unless you have a good reason, you should not set this to false.
  * **torrent-added-verify-mode:** String ("fast", "full", default: "fast") Whether newly-added torrents' local data should be fully verified when added, or wait and verify them on-demand later. See [#2626](https://github.com/transmission/transmission/pull/2626) for more discussion.
  * **utp-enabled:** Boolean (default = true) Enable [Micro Transport Protocol (µTP)](https://en.wikipedia.org/wiki/Micro_Transport_Protocol)
+ * **preferred-transport:** String ("utp" = Prefer µTP, "tcp" = Prefer TCP; default = "utp") Choose your preferred transport protocol (has no effect if one of them is disabled).
```

Comment:
> has no effect if one of them is disabled

I guess this would work but it feels a little sketchy to allow config settings that can contradict each other.

Just brainstorming., so feel free to push back -- what if we folded all three of these settings into a `preferred-transports` entry, where the default value would be `utp,tcp` and users could swap the order or remove one as they prefer.

Reply:
I'd like these 3 settings to combine eventually as well, but I didn't for now just to be 100% sure this feature is not going to be considered a breaking change.

Reply:
yeah, I guess it would be a breaking change.

Oh well, something for 5.0.0. :shrug: